### ISSUE TITLE
Frequency Lists Don't Work in 2.9.0 Preview 2, work fine in 2.8.9

### ISSUE THREAD

**Issue by JJMANJJ** [lrorpilla/jidoujisho#387]
Yomichan format frequency lists don't seem to work in 2.9.0 Preview 2. They install but don't show up. However, after uninstalling this app and reinstalling 2.8.9, I am able to successfully see the frequency lists just fine. I tested this behavior on multiple devices and with multiple different frequency lists with the same results.

**Comment by avc1657** [comment]
Yeah, I reported the same thing 2 months ago. Only a few frequency lists seem to be consistently working in 2.9.0 and that's one of the reasons I downgraded to 2.8.9.

**Comment by JJMANJJ** [comment]
> Yeah, I reported the same thing 2 months ago. Only a few frequency lists seem to be consistently working in 2.9.0 and that's one of the reasons I downgraded to 2.8.9.

I saw your post but I wasn't sure if it was the same issue since I couldn't get any of my 5 frequency lists to work. The release page says that 2.9.0 added Yomichan Strctured Content support. I imaging that thats what broke the frequency list support. I found that the frequency lists work if you import them in 2.8.9 and then upgrade to 2.9.0, you just can't import them directly to 2.9.0.

**Comment by avc1657** [comment]
Yeah 2.9.0 broke a lot of things as far as i can see. I got some JMDict formatting problems when exporting cards, half my frequency lists stopped working, recursive lookups is not broken, but way more difficult to execute, etc.
